src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpPost.java

```diff
@@ -73,14 +83,20 @@ public void ldpcvOfLdpnrsMustSupportPostWithoutMementoDatetimeHeader(final Strin
                                         ps);
 
         //same as previous test but with LDP-NR
+        // create the versioned resource
+        final String newResource = createVersionedNonRDFResource(uri, info).body().asString();
+        // create a memento for the current time
+        final Response versionResponse = doPost(newResource + "/fcr:versions");
+        confirmPresenceOfLinkValue(MEMENTO_LINK_HEADER, versionResponse);
+        confirmPresenceOfVersionLocationHeader(versionResponse);
```

Comment:
Per the requirement being tested, I would expect this test to also be verifying that the state of the created memento is indeed the same the current LDPRv.